tests/memray/memray_logger.py

```diff
@@ -0,0 +1,26 @@
+"""
+Create 100,000  logger instances and run a log test on them
+"""
+
+import picologging
+from io import StringIO
+
+
+def log(level=picologging.INFO):
+    logger = picologging.Logger("test", level)
```

Comment:
Just noting that actual user code shouldnt be instantiating directly like this, per the docs: https://docs.python.org/3/library/logging.html#logger-objects
I guess it should be expected to not produce memory leaks? The parent/child hierarchy won't work as expected, however.